test/convenience-method-test.js

```diff
@@ -706,6 +706,81 @@ module.exports = {
       test.done();
     }
   },
+  "#nextDates": {
+    "Correct number of dates are returned for RecurrenceRule": function(test) {
+      var job = schedule.scheduleJob({
+        second: null
+      }, function() {});
+
+      var dates = job.nextDates();
+      test.equal(dates.length, 1);
+
+      var dates = job.nextDates(5);
+      test.equal(dates.length, 5);
```

Comment:
It'd be great it you could check on every test that the dates are the expected ones (not just that the number of dates returned match)